ui/src/pages/Lore/lore.scss

```diff
@@ -0,0 +1,17 @@
+.lore {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  &__content {
+    @extend .lore;
+    width: 80%;
+  }
+}
+
+p {
+  margin: 0;
+}
+
+h3 {
+  margin-bottom: 0;
+}
```

Comment:
For these, dont do the stylings like this. Add classnames, this is bad practice

Reply:
this has been done lol xD